src/java.base/share/classes/java/lang/CharacterData00.java.template

```diff
@@ -25,12 +25,14 @@
 
 package java.lang;
 
+import jdk.internal.vm.annotation.Stable;
+
 /**
  * The CharacterData00 class encapsulates the large tables once found in
  * java.lang.Character
 */
 
-class CharacterData00 extends CharacterData {
+final class CharacterData00 extends CharacterData {
     /* The character properties are currently encoded into 32 bits in the following manner:
```

Comment:
Maybe also add the table `B` comment from `GenerateCharacter` to these comments: https://github.com/openjdk/jdk/blob/28f509317d477c5f4076658f9ae9995aa6c53631/make/jdk/src/classes/build/tools/generatecharacter/GenerateCharacter.java#L158-L171